File: El Paseo/Java/src/main/java/grupo2/model/Punto_Retiro.java
package grupo2.model;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToOne;

@Entity
public class Punto_Retiro {
	@Id @GeneratedValue
    @Column(name="PUNTORETIRO_ID")
    private Long id;
	private boolean borrado;
	private String nombre;
	@OneToOne 
	private Direccion direccion;
	
	public Punto_Retiro() {
		super();
	}

	public Long getId() {
		return id;
	}
	
	public boolean isBorrado() {
		return borrado;
	}

	public void setBorrado(boolean borrado) {
		this.borrado = borrado;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public Direccion getDireccion() {
		return direccion;
	}

	public void setDireccion(Direccion direccion) {
		this.direccion = direccion;
	}
	
	
}

File: El Paseo/Angular/src/app/modelos/punto-retiro.ts
import { Direccion } from "./direccion";

export class Punto_Retiro {
    public id:number;
    public borrado:boolean;
    public nombre:string;
    public direccion:Direccion;
    public constructor(id:number,borrado:boolean,nombre: string, direccion:Direccion){
        this.id = id;
        this.borrado = borrado;
        this.nombre = nombre;
        this.direccion = direccion;
    }
}
